packages/gatsby-theme-carbon/src/components/Switcher/Switcher.module.scss

```
@@ -36,7 +36,7 @@
     // https://css-tricks.com/using-css-transitions-auto-dimensions/
     // Tied to the height of the app switcher. Allows for animating up to auto.
     // Matching the height exactly is imperitive to keep the correct animation timing.
-    max-height: 458px;
+    max-height: 492px;
```

Looks like it's supposed to cut off 24px after the last item per #57 

```suggestion
    max-height: 508px;
```

Looking at a build from 2 weeks ago, it wasn't right even then (too short).